### DISCUSSION TITLE
Conditional Holograms

### DISCUSSION THREAD

**Discussion by VimaMT** [Zrips/CMI#4297] [Feature Requests]
It would be great to sometimes display holograms only to players who have permission to see those holograms.

For instance, let's say I want to set up a portal and provide access for that portal to one group of people. I also want to display a hologram, showing what that portal is ... but I only want that one group of people to see that hologram. From the perspective of everybody else, the hologram and the portal do not exist.

We already have the portal functionality. Can we please add the permissions to the hologram displays?

(Forgive me if you already have this functionality.)

**Comment by Gilmee** [comment]
This is already a feature. The lever icon in the gui when you go into that hologram's settings is to toggle permission requirement. 
![Screenshot_10](https://user-images.githubusercontent.com/69701603/95017571-053e8380-0652-11eb-9839-90a28d9cdfb6.png)

**Comment by VimaMT** [comment]
I like to place holograms in front of my portals. Does this help with the holograms?

**Comment by VimaMT** [comment]
Closing
